presto-native-execution/presto_cpp/main/types/PrestoToVeloxExpr.cpp

```diff
@@ -140,6 +140,7 @@ velox::variant VeloxExprConverter::getConstantValue(
     case TypeKind::REAL:
       return valueVector->as<velox::SimpleVector<float>>()->valueAt(0);
     case TypeKind::VARCHAR:
+    case TypeKind::VARBINARY:
```

Comment:
I did verify the following code in Velox:
template <TypeKind KIND>
struct VariantTypeTraits<
    KIND,
    std::enable_if_t<
        KIND == TypeKind::VARCHAR || KIND == TypeKind::VARBINARY,
        void>> {
  using native_type = folly::StringPiece;
  using stored_type = scalar_stored_type<KIND>;
};

so this is fine indeed.